crates/bootstrap/src/lib.rs

```diff
@@ -3,14 +3,15 @@ mod custom_texture;
 use std::fmt::Write;
 use std::time::Instant;
 
-use winit::dpi::LogicalSize;
-use winit::event::{Event, WindowEvent};
-use winit::event_loop::{ControlFlow, EventLoop};
-use winit::window::WindowBuilder;
+use winit::application::ApplicationHandler;
+use winit::event::WindowEvent;
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
 
+use winit::window::{Window, WindowAttributes, WindowId};
```

Comment:
I think we might've messed up the grouping a smidge here, let's regroup the winit imports!